### PR TITLE
docs: comments

### PR DESCRIPTION
**Pull request by abhinavr4**

### CODE REVIEW

File: collaboration-and-sharing/README.md [hunk 1]
```diff
@@ -133,8 +133,10 @@ The table below summarises the operations that can be performed on media library
 | Download file                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | Download assets as zip           | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
 | View downloads                   | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| View comments                    | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
-| Edit and delete comments         |                    | :heavy_check_mark: | :heavy_check_mark: |
+| View threads and comments        | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: |
+| Add new comment and replies      |                    | :heavy_check_mark: | :heavy_check_mark: |
+| Edit and delete own comments     |                    | :heavy_check_mark: | :heavy_check_mark: |
+| Modify own reaction on comments  |                    | :heavy_check_mark: | :heavy_check_mark: |
```
Comment (aman051197) [inline collaboration-and-sharing/README.md:139]: What about resolving threads?

Comment (abhinavr4) [reply collaboration-and-sharing/README.md:139]: Added resolve. Delete was already there in the list.

File: media-library/overview/comments-on-assets.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+- **Copy and share Link:** Users have the ability to copy and share links to specific threads. This allows them to easily reference discussions or share them with relevant stakeholders.
+
+- **Emoji reactions:** Users can react to comments using emojis, providing a quick and expressive way to acknowledge or respond to comments without adding additional text.
+
+- **Rich text editor:** ImageKit offers a rich text editor, empowering users to enhance their comments with formatting options such as bold, italic, strikethrough, adding links, quoting, and creating bulleted or numbered lists.
+
+- **Emojis in comments:** Users can include emojis within their comments to add expressiveness and convey emotions.
+
+- **Mention other users:** A user can mention other users within the account in their comments, drawing their attention to specific discussions or seeking their input. This fosters collaboration and ensures that relevant stakeholders are engaged in the conversation.
```
Comment (aman051197) [inline media-library/overview/comments-on-assets.md]: Users with access to media-library, not within the account. Right?

Comment (abhinavr4) [reply media-library/overview/comments-on-assets.md]: Specified this

File: media-library/overview/comments-on-assets.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+- **Watcher Definition:** Watchers of a thread include the owner of the file, anyone participating in the thread by commenting, and users mentioned within the thread discussion. This ensures that relevant stakeholders are kept informed of ongoing discussions and developments.
+
+- **Email Notifications:** Watchers receive email notifications for the following types of new activity on a thread:
+  - Creation of a new thread with comment.
+  - Replies on a thread, facilitating ongoing conversation.
+  - Deletion of a thread.
+  - Resolution or unresolution of a thread, signaling changes in status or resolution.
+
+  Notifications are only sent to users who have at least "Can view" permission on the file.
```
Comment (aman051197) [inline media-library/overview/comments-on-assets.md]: Should we mention that user performing the action will not receive an email regarding it?

Comment (abhinavr4) [reply media-library/overview/comments-on-assets.md]: Added

**Comment by aman051197** [comment]
I can see the modHeader icon in all the images. Can we remove it?

File: media-library/overview/comments-on-assets.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  - Creation of a new thread with comment.
+  - Replies on a thread, facilitating ongoing conversation.
+  - Deletion of a thread.
+  - Resolution or unresolution of a thread, signaling changes in status or resolution.
+
+  Notifications are only sent to users who have at least "Can view" permission on the file. A user won't receive notifications for their own actions.
+
+## Access control and permissions
+
+Read about the permission levels required for restricted media library users for different comment operations [here](../../collaboration-and-sharing/README.md#file-and-folder-permission-levels).
```
Comment (aman051197) [inline media-library/overview/comments-on-assets.md]: Can add link to restricted media library users page.

Comment (abhinavr4) [reply media-library/overview/comments-on-assets.md]: Added

**Pull request merged**